libs/components/i18n/src/lib/modules/i18n/locale-provider.ts

```diff
@@ -4,15 +4,13 @@ import { Observable, of as observableOf } from 'rxjs';
 
 import { SkyAppLocaleInfo } from './locale-info';
 
+const SKY_APP_LOCALE_PROVIDER_DEFAULT_LOCALE = 'en-US';
+
 @Injectable({
   providedIn: 'root',
 })
 export class SkyAppLocaleProvider {
-  public get defaultLocale(): string {
-    return SkyAppLocaleProvider._defaultLocale;
-  }
-
-  private static _defaultLocale = 'en-US';
+  public defaultLocale: string = SKY_APP_LOCALE_PROVIDER_DEFAULT_LOCALE;
```

Comment:
I think this should remain as a `get` accessor. This change allows a consumer to now set `defaultLocale`.

Reply:
Good catch. Fixed.